client-go/README.md

```diff
@@ -0,0 +1,304 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+To use go client, add "import github.com/yanhongwangg/incubator-iotdb/client-go" 
+
+## Requirement
+* GoLang 1.14+
+
+## Getting started
+*Create project eg:session
+
+*Enter project directory eg:cd session
+
+*Download dependency : go get github.com/yanhongwangg/incubator-iotdb@client-go
+
+*eg:session_example.go
+
+```
+package main
+
+import (
```

Comment:
if those codes same as the session_example.go, I think we can just add the session_example.go link here.

Reply:
It's exactly the same. I'll post the link